src/app/components/core/control-dispatcher/list-control-dispatcher/list-control-dispatcher.component.ts

```diff
@@ -34,6 +39,69 @@ export class ListControlDispatcherComponent implements OnInit {
       .subscribe(viewport => {
         this.viewPort = viewport;
       });
+    this.getInstalledServicesList();
+  }
+
+  public getInstalledServicesList() {
+    /** request start */
+    this.ngProgress.start();
+    this.servicesApiService.getInstalledServices().
+      then((data: any) => {
+        /** request done */
+        this.ngProgress.done();
+        // Check if dispatcher service available in installed services list
+        this.dispatcherServiceInstalled = data.services.includes('dispatcher');
+        if (this.dispatcherServiceInstalled) {
+          this.checkServiceStatus();
```

Comment:
check schedules 

if not exists - not added
if exists and not enabled then - not enabled